app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @chef = Chef.find(params[:chef_id])
+  end
+end
```

Comment:
Take this with a grain of salt but I am personally not a fan of this. What I would have probably done would have been to do something like this:

```ruby
def index
  chef = Chef.find(params[:chef_id])
  @ingredients = chef.ingredients
end
```

This way in my view I would just be able to iteration through `@ingredients` and It wouldn't matter in the controller if it was `Ingredient.all` or `chef.ingredients` or whatever because the view wouldn't need to change. I personally thing is weird for the `index` action of the `IngredientsController` to be sending a `@chef` to the view. Again though, personal opinion and you can take it with a grain of salt.